Sources/App/Utilities/JWT/JWTMiddleware.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  JWTMiddleware.swift
+//  App
+//
+//  Created by Arkadiusz Żmudzin on 13/03/2020.
+//
+
+import Vapor
+import JWT
+
+class JWTMiddleware: Middleware {
+    
+    func respond(to request: Request, chainingTo next: Responder) throws -> EventLoopFuture<Response> {
+        let token = request.token
+        if !token.isEmpty {
```

Comment:
This feels more swifty to me... 
```
guard !token.isEmpty else {
      throw Abort(.unauthorized, reason: "No Access Token")
}
```

What do you think?

Reply:
Sure, as you can clearly see, I forgot about `guard` syntax :D